gax/src/main/java/com/google/api/gax/rpc/ServerStreamingAttemptCallable.java

```diff
@@ -335,14 +335,25 @@ private void onRequest(int count) {
 
   /** Called when the inner callable has responses to deliver. */
   private void onAttemptResponse(ResponseT message) {
+    message = resumptionStrategy.processResponse(message);
+
+    // ResumptionStrategy suppressed the message
+    if (message == null) {
+      // Request the next one and exit
+      if (!autoFlowControl) {
+        innerController.request(1);
```

Comment:
What does this do? Also looks suspicious, since it does not do same what should happen if  `!autoFlowControl` when `message != null` (check line 352).